### PR TITLE
Update PSS UI to Show Seperate EU/t Values

### PR DESCRIPTION
**Pull request by Ghostipedia**
## What
Updates The PSS UI to Split the EU In and Out Into seperate values
Before : 
image
After : 
image

_Linking an issue can be used alternatively to writing a description._

## Implementation Details
This is a Direct Port of @serenibyss 's changes from 1.12.2 
https://github.com/GregTechCEu/GregTech/commit/b4f7e1702cdc046dc074ef545d2e1e7791297432

## Additional Information
There was some extra Datagen that generated alongside what I was working with, I've committed it just in case - if not feel free to axe those files.

## Potential Compatibility Issues
Shouldn't be any incompatibilities

### CODE REVIEW

File: src/generated/resources/assets/gtceu/lang/en_us.json [hunk 1]
```diff
@@ -1602,14 +1602,12 @@
   "compass.node.gtceu.materials/block": "Block",
   "compass.node.gtceu.materials/bolt": "Bolt",
   "compass.node.gtceu.materials/chainsaw_head": "Chainsaw Head",
-  "compass.node.gtceu.materials/chipped_gem": "Chipped Gem",
```
Comment (screret) [inline src/generated/resources/assets/gtceu/lang/en_us.json]: Whoops! You have to enable this config for dev.

File: src/generated/resources/assets/gtceu/lang/en_us.json [hunk 2]
```diff
@@ -1602,14 +1602,12 @@
...
   "compass.node.gtceu.materials/crushed_ore": "Crushed Ore",
   "compass.node.gtceu.materials/dense_plate": "Dense Plate",
   "compass.node.gtceu.materials/double_plate": "Double Plate",
   "compass.node.gtceu.materials/dust": "Dust",
   "compass.node.gtceu.materials/exquisite_gem": "Exquisite Gem",
   "compass.node.gtceu.materials/fine_wire": "Fine Wire",
-  "compass.node.gtceu.materials/flawed_gem": "Flawed Gem",
```
Comment (screret) [inline src/generated/resources/assets/gtceu/lang/en_us.json]: ditto

**Review by screret — CHANGES_REQUESTED** [review]

File: src/generated/resources/assets/gtceu/lang/en_us.json (hunk 1 above)
Comment (Ghostipedia) [reply src/generated/resources/assets/gtceu/lang/en_us.json]: Sorted Ty

File: src/generated/resources/assets/gtceu/lang/en_us.json (hunk 2 above)
Comment (Ghostipedia) [reply src/generated/resources/assets/gtceu/lang/en_us.json]: Sorted Mk2

**Review by screret — APPROVED** [review]

**Pull request merged**